src/components/ReminderButton/ReminderButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
Anton, your current implementation is not following Material UI guidelines, the button is indeed to big for example.
Please take a look at the example available in the offical documentation. [AppBar docs](https://material-ui.com/components/app-bar/#app-bar-with-a-primary-search-field)